src/blocks/posts/posts.ts

```diff
@@ -1,6 +1,13 @@
 import { html, render } from 'lit';
 import { createOptimizedPicture } from '../../utils/createOptimizedPicture';
+import { fetchData } from '../../utils/fetchData';
 
+interface SheetsResponse<T> {
```

Comment:
- auslagern in eigene "shared.types.ts" 
- umbennen in SitemapResponse
- prüfen ob wir den generic T brauchen